tests/default.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_pipeline {
```

Comment:
@TCLamnidis I'll leave this for you to update - basically just copy over your exisitng `test.nf.test` into here

Then we need to check all the other tests are working (e.g. adding the `profile` part to each header)